src/styles/App.scss

```diff
@@ -173,14 +227,14 @@ button {
         margin: 0;
   
         .my-name {
```

Comment:
Porque my-name e my-page se repetem o tempo todo?? porque não colocar o font-family para h1, h2, h3 etc... ?

Reply:
Oh, nice hint! With the responsible functionality I was just copy and pasting these classes to be overwitten to each "@media"; but it's really unnecessary.